frontend/src/js/store/releasesSlice/index.ts

```diff
@@ -15,9 +15,38 @@
 import { DEVICE_LIST_DEFAULTS, SORTING_OPTIONS } from '@northern.tech/store/constants';
 import { createSlice } from '@reduxjs/toolkit';
 
+import { Release } from '../api/types/Release';
+
 export const sliceName = 'releases';
 
-export const initialState = {
+interface SortOptions {
+  direction?: string;
```

Comment:
I think it should be narrowed to possible directions

Reply:
right... I reused one of the existing type definitions for the same in the codebase, but in https://github.com/mendersoftware/mender-server/pull/460/commits/246e5ba5d8608db264a317a8a390ca6a6f12c32e I made this dependent on the directions... later I would put this in the `types` package we talked about (to decouple this from the store)